### DISCUSSION TITLE
How to access the database

### DISCUSSION THREAD

**Discussion by StonzX** [Ansarimajid/College-ERP#13] [Q&A] [answered]
Can u tell me how to access the database and is the data saved on server an how to access the data

**Comment by Ansarimajid** [comment, marked as answer]
> Can u tell me how to access the database and is the data saved on server an how to access the data

You can connect and access database in settings.py where you can paste your database configuration which may PostgreSQL, MySQL etc, In this project I used local database SQLite which is a simple local database that can be used to store data.

If you're facing any issue, you can open Issue I will try to resolve as soon as possible..
